docs/developer/eventing/sources/kafka-source/README.md

```diff
@@ -13,63 +6,80 @@ Tutorial on how to build and deploy a `KafkaSource` event source.
 
 The `KafkaSource` reads all the messages, from all partitions, and sends those messages as CloudEvents via HTTP to its configured `sink`.
 
-*NOTE:* In case you need a more sophisticated Kafka Consumer, with direct access to specific partitions or offsets you might want to implement a _Kafka Consumer_, using one of the available Apache Kafka SDKs, to handle the messages yourself, rather than using the Knative `KafkaSource`.
+!!! note
+    If need a more sophisticated Kafka Consumer, with direct access to specific partitions or offsets, you can implement a Kafka Consumer by using one of the available Apache Kafka SDKs.
 
-## Prerequisites
+<!--TODO: Check if this note is out of scope; should we not mention anything beyond the direct Knative/Kafka integration we provide?-->
 
-- A Kubernetes cluster with [Knative Kafka Source installed](../../../admin/install/).
+## Installing the Kafka event source CRD
 
-## Apache Kafka Topic (Optional)
+Install the Kafka event source by running the command:
 
-1. If using Strimzi, you can set a topic modifying `source/kafka-topic.yaml`
-   with your desired:
+```bash
+kubectl apply -f https://storage.googleapis.com/knative-nightly/eventing-kafka/latest/source.yaml
```

Comment:
This link is targeting nightly builds. We should probably point people to the GitHub releases instead.

Reply:
@antoineco would you be able to provide a command that works please? 🙂 
Thanks for calling this out, I had a note here to check what the right command would be because I wasn't sure, and this was what's available in the kafka sandbox-repo md file

Reply:
I was thinking about the release manifests you can find here: https://github.com/knative-sandbox/eventing-kafka/releases/

If you look at the latest release (0.25) you'll see 2 assets related to the source:
- source-crds.yaml
- source.yaml

Those should be applied using `kubectl apply`.

Note: Because the "latest" version will change with every Eventing release, it's probably not wise to put a direct link in the docs, because eventually the documented link will rot (point to an old version). How it this done for serving and eventing? Are versions manually updated in the docs on every release?